### PR TITLE
edit pass: change-azure-subscription

### PR DESCRIPTION
**Pull request by ShawnJackson**
@chcomley, @kasangi – Please review the copy edit to this article and let me know if I’ve changed the technical meaning anywhere. This is for VSTS work request 1302753.

Two of the images in the "Remove the billing subscription" section aren't showing up. Have those images moved to a different folder? If so, I'll update the paths in the article.

Right before the "Remove the billing subscription" section, there's a "Remove the billing subscription" link to that section. This link doesn't seem to serve a purpose. I recommend deleting it.

### CODE REVIEW

**Comment by ShawnJackson** [comment]
Fixed a merge conflict.

File: docs/organizations/billing/change-azure-subscription.md [hunk 1]
```diff
@@ -16,59 +16,61 @@ ms.date: 07/10/2018
...
+### Remove the billing subscription
 
 1. [Sign in to the Azure portal](https://portal.azure.com/) as VSTS organization owner and as Azure subscription co-administrator or greater.
 
-    If you experience browser problems with Azure,
-    make sure that you use a [supported browser](/azure/azure-preview-portal-supported-browsers-devices).
+   If you experience browser problems with Azure, make sure that you use a [supported browser](/azure/azure-preview-portal-supported-browsers-devices).
 
-2. Go to **All services** > **Team Services organizations**. 
+2. Go to **All services** > **Team Services accounts**.
```
Comment (chcomley) [inline docs/organizations/billing/change-azure-subscription.md]: I'll have to go back in and fix this any day, but for now, can we please leave this as Team Services "administration"? This branch isn't reflecting the latest update, where it's likely to be a bug, but I've aligned with the current Azure portal, which displays Team Services administration. So, line 59 and 61 changes can be removed.

File: docs/organizations/billing/change-azure-subscription.md [hunk 2]
```diff
@@ -16,59 +16,61 @@ ms.date: 07/10/2018
...
+   If you experience browser problems with Azure, make sure that you use a [supported browser](/azure/azure-preview-portal-supported-browsers-devices).
 
-2. Go to **All services** > **Team Services organizations**. 
+2. Go to **All services** > **Team Services accounts**. 
 
-   ![Choose All services and Team Services organizations](_img/change-azure-subscription/all-services-team-services-organizations.png)
+   ![Choose All services and Team Services accounts](_img/change-azure-subscription/all-services-team-services-organizations.png)
 
-3. Select your organization and remove billing.
+3. Select your organization and remove billing by selecting **Unlink**.
```
Comment (chcomley) [inline docs/organizations/billing/change-azure-subscription.md]: Please omit this change, too. The portal no longer says "unlink" and master is correct

File: docs/organizations/billing/change-azure-subscription.md [hunk 3]
```diff
@@ -16,59 +16,61 @@ ms.date: 07/10/2018
...
+* [VSTS project collection administrator or organization owner permissions](../accounts/faq-add-delete-users.md#find-owner)
+* [The owner or contributor role on your Azure subscription](add-backup-billing-managers.md)
 
 >[!NOTE]
-> When you remove the billing subscription from your organization, any paid quantities of VSTS Basic, Package Management users, Test Manager users, Microsoft-hosted Pipelines, and self-hosted Pipelines you’ve paid for this month will continue uninterrupted until the 1st of next month, but your organization will revert immediately to the Free Tier for cloud-based load testing. Removing the subscription will also cancel any non-Microsoft paid extensions without refund or credit.
+> When you remove the billing subscription from your organization, any paid quantities of VSTS Basic, Package Management users, Test Manager users, Microsoft-hosted pipelines, and self-hosted pipelines you’ve paid for this month will continue uninterrupted until the first of next month. But your organization will revert immediately to the free tier for cloud-based load testing. Removing the subscription will also cancel any non-Microsoft paid extensions without refund or credit.
```
Comment (chcomley) [inline docs/organizations/billing/change-azure-subscription.md]: Can we please confirm whether "pipelines" should be capitalizated? Karthik?

File: docs/organizations/billing/change-azure-subscription.md (hunk 2 above)
Comment (ShawnJackson) [reply docs/organizations/billing/change-azure-subscription.md]: I reverted this change.

**Pull request merged**